dashboard/db/migrate/20241202234601_add_index_to_school_infos_country_and_school_name_and_school_type.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToSchoolInfosCountryAndSchoolNameAndSchoolType < ActiveRecord::Migration[6.1]
+  def change
+    add_index :school_infos, [:school_name, :country, :school_type]
```

Comment:
Very possible this is just Rails magic but I see multiple indices added in the annotations and schema.rb but only one here?

Reply:
oh hmm... yeah let me take another look. I had previously run a migration that added an index to just country and school_name, and tried to undo the migration and re-apply it with school_type 😬

Reply:
okay I did a reset of my local db off of `staging-next` and re-ran the migration, so there's only the one added index 😅